up-l1/mqtt.adoc

```diff
@@ -35,8 +35,128 @@ MQTT is an OASIS standard messaging protocol for the Internet of Things (IoT). I
 
 For more information, please refer to https://mqtt.org/
 
-
 == Specifications
 
+=== UAttribute Mapping
+
+MQTT 5 supports custom header fields, and we leverage this to map the UAttribute values into the MQTT header.
+
+[cols="1,1"]
+|===
+| MQTT Header Property Code | Value (key:value)
+| UserProperty
```

Comment:
In order to drastically save space (each byte is critical), lets use the protobuf defined field numbers in lieu of string names so that it is easier to convert and takes less space.

Reply:
Alright, I updated the table to use the protobuf defined field numbers. One thing to note, the UserProperty is still a Utf8 encoded string pair in the MQTT 5 specification.